actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -25,9 +25,15 @@ def cache_collection_render(instrumentation_payload)
         end
       end
 
+      def callable_cache_key?
+        @options[:cached].respond_to?(:call)
+      end
```

Comment:
Let's inline this helper. I know I did wrote it like this originally, but I since came to view it as an overabstraction 😁